synapse_data_warehouse/synapse_raw/tables/V2.20.0__create_accessrequirementsnapshots_table.sql

```diff
@@ -0,0 +1,87 @@
+USE SCHEMA {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP
+USE WAREHOUSE COMPUTE_MEDIUM;
+
+CREATE OR ALTER TABLE ACCESSREQUIREMENTSNAPSHOTS (
+    CHANGE_TIMESTAMP TIMESTAMP_NTZ(9) COMMENT 'The time when the change (created/updated/deleted) on an access requirement is pushed to the queue for snapshotting.',
```

Comment:
Just asking here, but does SNOWFLAKE itself auto configure these values? Is it "better" to use `TIMESTAMP` instead of "TIMESTAMP_NTZ(9)"?

Reply:
I think the main difference is that TIMESTAMP keeps track of the timezone information and TIMEZONE_NTZ does not. I was just going off of the [example](https://sagebionetworks.jira.com/wiki/spaces/DPE/pages/3109683240/Contributing) in the confluence page so I used NTZ. Since all of this raw synapse data is coming from one source, do we need to keep track of timezones?